src/components/Shared/StyledTableCell.tsx

```diff
@@ -0,0 +1,15 @@
+import { styled } from '@mui/material/styles';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+
+const StyledTableCell = styled(TableCell)(() => ({
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+  },
+  '&:first-of-type': {
+    width: '50px',
+  },
+  paddingTop: '10px',
+  paddingBottom: '10px',
+}));
+
+export default StyledTableCell;
```

Comment:
I don't agree with creating new components just to add styles. I realize it does this in the docs examples, but we have a theme already, and I'd prefer to add this there.

[example for table styles](https://gist.github.com/kimberlythegeek/2d5806d660330f600b4930f2f675ead4)

and you can set the default body font size to 14 in `typography.js`: `fontSize: 14` below the `fontFamily` attribute